flask_sqlalchemy/cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+    flask_sqlalchemy.cli
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Command Line Interface for managing the database.
+
+    :copyright: (c) 2017 by David Baumgold.
+    :license: MIT, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+from functools import wraps
+
+import click
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+try:
+    from flask.cli import with_appcontext
+except ImportError:
+    from flask_cli import with_appcontext
+
+state = LocalProxy(lambda: current_app.extensions['sqlalchemy'])
+
+
+def commit(fn):
+    """Decorator to commit changes after the command is run."""
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        fn(*args, **kwargs)
+        state.db.session.commit()
+    return wrapper
+
+
+@click.group()
+def db():
+    """Manages the database."""
+
+
+@db.command('create')
+@click.option('--bind', default='__all__')
+@with_appcontext
+@commit
+def db_create(bind):
+    """Creates database tables."""
+    state.db.create_all(bind=bind)
+    click.secho('Database created successfully.', fg='green')
+
+
+@db.command('drop')
+@click.option('--bind', default='__all__')
+@click.confirmation_option(prompt=(
+    "This will destroy all the data in your database. "
```

Comment:
I see where you're coming from -- this is a potential footgun, and we should be careful with it. At the same time, I can imagine circumstances where it has a use with debug mode turned off. For example, it's reasonable to have an automated test flow where the code is automatically checked out, setup, run, and torn down, and where debug mode is specifically turned off, in order to make the test more consistent with a production environment. There's no reason not to support that flow, and I think that having a confirmation step will deter most human error.